en/flight_modes/README.md

```diff
@@ -18,16 +18,12 @@ The sections below provide a high-level overview of the modes (more detail can b
 **Fixed wing aircraft:**
 
 - **Manual**: The pilot has direct control (via the safety
-  coprocessor) if the ACRO switch is unmapped or off. This is the only
+  coprocessor). This is the only
   mode which overrides the FMU and it provides a safety mechanism
   which allows full control of throttle, elevator, ailerons and rudder
   via RC in the event of an FMU firmware malfunction.
-- **Stabilized (aka fly-by-wire)**: If the ACRO switch is ON, the
-  pilot's inputs are passed as roll and pitch *angle* commands and
-  direct rudder control. If Sticks are released the Aircraft will
-  level out. Drift from Wind or other sources will not be compensated.
-  This mode requires the FMU to be operating properly. Also note that
-  the ACRO switch has essentially the opposite effect for multirotors.
+- **Stabilized (aka fly-by-wire)**: The pilot's inputs are passed as roll and pitch angle commands to the autopilot and as yaw command directly to the output mixer to control rudder manually. If the RC roll and pitch sticks are centered, the autopilot regulates the roll and pitch angles to zero, hence stabilizing (leveling-out) the attitude against any wind disturbances. However, in this mode the position of the aircraft is not controlled by the autopilot, hence the position can drift due to wind. With nonzero roll, pitch and yaw inputs the vehicle does a coordinated turn which has zero sideslip (the acceleration in y-direction (sidewards) is zero). During a coordinated turn, the rudder is used to control the sideslip.

```

Comment:
See my comment in Devguide: https://github.com/PX4/Devguide/pull/474#discussion_r171715031